packages/types/src/chip/index.ts

```diff
@@ -79,3 +79,59 @@ export const ChipTapResponseSchema = z.object({
 });
 
 export type ChipTapResponse = z.infer<typeof ChipTapResponseSchema>;
+
+
+export const UpdateLeaderboardEntryRequestSchema = z.object({
+  authToken: z.string(),
+  chipIssuer: ChipIssuerSchema,
+});
+
+export type UpdateLeaderboardEntryRequest = z.infer<
+  typeof UpdateLeaderboardEntryRequestSchema
+>;
+
+export const GetLeaderboardEntryRequestSchema = z.object({
+  authToken: z.string(),
+  chipIssuer: ChipIssuerSchema,
+});
+
+export type GetLeaderboardEntryRequest = z.infer<
+  typeof GetLeaderboardEntryRequestSchema
+>;
+
+export const LeaderboardEntrySchema = z.object({
+  username: z.string(),
+  chipIssuer: z.string(),
+  tapCount: z.number(),
+  // In the future could imagine name, chip fields being added to allow custom leaderboards corresponding to specific chips
```

Comment:
Picturing a world where we have more leaderboards I imagine all we'd need to do is add a name field for the specific leaderboard the entry corresponds to.